### ISSUE TITLE
Dynamic Height for AG Grid

### ISSUE THREAD

**Issue by safeerk** [ceolter/ag-grid#1484]
Dears, 
         Recently i tried to update the Ag grid from 5.2 to 8.0.1 after that my ag grid doest take auto height .
Before i use to give height as 100 % it works like charm. Now i need to manually set the height on pixel.

From the code docs i found out that atleast the container class has to be a fixed , though i have set up it.

**Code:**
 <div id="accounts" class="ag-blue" style="width:99%; padding-left:15px;height:100%;"></div>

**Before(v5.2):**
![image](https://cloud.githubusercontent.com/assets/22815639/23155222/c93c7cac-f822-11e6-9735-49fa188e72f3.png)

**Now(v8.0.1):**

![image](https://cloud.githubusercontent.com/assets/22815639/23155308/2253eb9a-f823-11e6-9614-53e04e572689.png)

In v8.0.1 if i set the manual Height to 200 px instead of 100% it will come:
![image](https://cloud.githubusercontent.com/assets/22815639/23155331/4c1ba846-f823-11e6-9958-a812e524d4b2.png)

Kindly guide me on this Thanks on advance!!!..

**Comment by seanlandsman** [comment]
If you're using DOCTYPE html you will need to set the parent container (typically a div) to a height of some value. If you do this the grid will work with a % as before.

This may help you http://stackoverflow.com/questions/10871898/why-cant-i-make-my-div-100-height-if-i-use-an-html5-doctype-how-do-i-get-it-1

**Comment by lisaanjuli** [comment]
I have tried this, but in my case fixed size of parent container does not fix my problem.

In my application the requirement is that the user can minimize the screen. So when I set the parent container to a height of some value (for example 768px) then the table will not size up to the maximum.

Is there any solution for that?

When I do not set a value for the parent container and implement in the div of the table width:100%, height:100% and position:fixed, the table does what I need but then the table is too big for the screen. When I set the width to 90% and the height to 95% it is better, but the table header is not shown correct. (and I do not think this should be the solution ;) )
